contents/handbook/people/team-structure/product-analytics/objectives.md

```diff
@@ -0,0 +1,91 @@
+Input from the execs:
+
+> - Objective: Get 80% of active b2b customers to add their marketing website
+>   - Why? Increase # events coming in, which increases revenue. Our hypothesis is that marketing websites have 10-100x more events than applications.
+> - Objective: Implement most frequently requested features with 3+ happy companies for the feature launch
+>   - Key results:
+>     - Implement the following:
+>       - Multiple breakdowns
+>       - Cohort style queries (did x event x times), but within a query itself
+>       - Composite events: ‘Any event’, ‘New user’ (users first event), ‘top events’
+>       - User composition (ie pie chart where you can select properties)/
+>       - Superscale querying - Get 98% of each company’s insights P99 below an acceptable time (to be worked out by the team)
```

Comment:
This is an objective/team priority on its own. If we're keen on exploring options such as getting marketing websites instrumented, getting customers larger than mid market, and/or growing with our current users, I'd suggest we keep high scale querying as a main top priority. There's the obvious point of making sure new features are performant but also some of the base existing functionality could use some 🧠 (ex: https://github.com/PostHog/posthog/issues/8981). 

There's optimizations that can be explored across the board (https://github.com/PostHog/posthog/issues?q=is%3Aissue+is%3Aopen+optimize+label%3Aperformance) and especially now that person on events is coming to a close, there's possibly more options to consider.

Reply:
Thanks @EDsCODE I've added this as a goal